### PR TITLE
Factor comoving magnetic field into free function, add hydro StressEnergy functions

### PR DESCRIPTION
**Pull request by nilsvu**
### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Review by nilsdeppe — DISMISSED** [review]

**Comment by nilsvu** [comment]
Added two commits with a few more pointwise functions

File: src/PointwiseFunctions/Hydro/StressEnergy.hpp [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+ * \brief The total mass-energy density measured by a normal observer, $E = n_a
+ * n_b T^{ab}$
+ *
+ * This quantity sources the gravitational field equations in the 3+1
+ * decomposition (see Eq. (2.138) in \cite BaumgarteShapiro).
+ *
+ * Perfect fluid contribution (Eq. (5.33) in \cite BaumgarteShapiro):
+ *
+ * \begin{equation}
+ *   E_\mathrm{fluid} = \rho_0 h W^2 - P
```
Comment (nilsdeppe) [inline src/PointwiseFunctions/Hydro/StressEnergy.hpp]: I don't know what rho_0 is or what P is. We use `rho` for the baryon rest mass density and `D` for `rho W`, where `W` is the Lorentz factor. We use lower case `p` for the pressure.

File: src/PointwiseFunctions/Hydro/StressEnergy.hpp [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+/*!
+ * \brief The spatial momentum density $S^i = -\gamma^{ij} n^a T_{aj}$
+ *
+ * This quantity sources the gravitational field equations in the 3+1
+ * decomposition (see Eq. (2.138) in \cite BaumgarteShapiro).
+ *
+ * Perfect fluid contribution (Eq. (5.34) in \cite BaumgarteShapiro):
+ *
+ * \begin{equation}
+ *   S^i_\mathrm{fluid} = \rho_0 h W^2 v^i
```
Comment (nilsdeppe) [inline src/PointwiseFunctions/Hydro/StressEnergy.hpp]: same notation comment here

File: src/PointwiseFunctions/Hydro/StressEnergy.hpp [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+ * \brief The trace of the spatial stress tensor, $S =
+ * \gamma^{ij}\gamma_{ia}\gamma_{jb}T^{ab}$
+ *
+ * This quantity sources the gravitational field equations in the 3+1
+ * decomposition (see Eq. (2.138) in \cite BaumgarteShapiro).
+ *
+ * Perfect fluid contribution (Eq. (5.36) in \cite BaumgarteShapiro):
+ *
+ * \begin{equation}
+ *   S_\mathrm{fluid} = 3 P + \rho_0 h (W^2 - 1)
```
Comment (nilsdeppe) [inline src/PointwiseFunctions/Hydro/StressEnergy.hpp]: same notation comment here

**Review by nilsdeppe — CHANGES_REQUESTED** [review]

**Review by nilsdeppe — APPROVED** [review]

**Pull request merged**